jaypeasee/national-parkfinder
# Feature/display parks

## Is this a fix or a feature?
Feature

## What is the change?
+ Fetch API data and set state of the `ParkContainer` and display unique info on page on load and by clicking a park button in the side navigation
+ Set up TypeScript interfaces to match the data of the API and pass those props to child components

## Where should the reviewer start?
+ `ParkContainer.tsx` & child components of `Banner` & `ParkInfo`
+ `App.tsx`

## How should this be tested?
+ See component dev tools when loading the page and by clicking on different park names
+ Reload the page to ensure a random park is generating 

## Relevant Tickets
+ #4 
+ #1
+ #5

### Gif
![](https://media.giphy.com/media/QZWWJmFBXOcMZc3Wk6/giphy.gif)

@saraho1123
@jaypeasee
@baileydunning
